### PR TITLE
MINOR: Fixed broken links in the documentation

### PR DESCRIPTION
**Pull request by vahidhashemian**

### CODE REVIEW

File: docs/design.html [hunk 1]
```diff
@@ -327,8 +327,12 @@
 <p>
 This majority vote approach has a very nice property: the latency is dependent on only the fastest servers. That is, if the replication factor is three, the latency is determined by the faster slave not the slower one.
 <p>
-There are a rich variety of algorithms in this family including ZooKeeper's <a href="http://www.stanford.edu/class/cs347/reading/zab.pdf">Zab</a>, <a href="https://ramcloud.stanford.edu/wiki/download/attachments/11370504/raft.pdf">Raft</a>,
-and <a href="http://pmg.csail.mit.edu/papers/vr-revisited.pdf">Viewstamped Replication</a>. The most similar academic publication we are aware of to Kafka's actual implementation is <a href="http://research.microsoft.com/apps/pubs/default.aspx?id=66814">PacificA</a> from Microsoft.
+There are a rich variety of algorithms in this family including ZooKeeper's
+<a href="http://web.archive.org/web/20140602093727/http://www.stanford.edu/class/cs347/reading/zab.pdf">Zab</a>,
```
Comment (vahidhashemian) [inline docs/design.html:331]: For two broken links that I couldn't find a live URL I used the latest version of the broken URL on Web Archive. If alternative URLs exist I could use them instead. This is one of those instances.

File: docs/ops.html [hunk 2]
```diff
@@ -566,7 +566,9 @@
 
 In Linux, data written to the filesystem is maintained in <a href="http://en.wikipedia.org/wiki/Page_cache">pagecache</a> until it must be written out to disk (due to an application-level fsync or the OS's own flush policy). The flushing of data is done by a set of background threads called pdflush (or in post 2.6.32 kernels "flusher threads").
 <p>
-Pdflush has a configurable policy that controls how much dirty data can be maintained in cache and for how long before it must be written back to disk. This policy is described <a href="http://www.westnet.com/~gsmith/content/linux-pdflush.htm">here</a>. When Pdflush cannot keep up with the rate of data being written it will eventually cause the writing process to block incurring latency in the writes to slow down the accumulation of data.
+Pdflush has a configurable policy that controls how much dirty data can be maintained in cache and for how long before it must be written back to disk.
+This policy is described <a href="http://web.archive.org/web/20160518040713/http://www.westnet.com/~gsmith/content/linux-pdflush.htm">here</a>.
```
Comment (vahidhashemian) [inline docs/ops.html:570]: And this is the other one.

**Comment by vahidhashemian** [comment]
@hachikuji More doc fixes, this time on broken links. Thanks for taking a look.

**Comment by hachikuji** [comment]
@vahidhashemian Thanks for the fixes. We had a separate issue for the varnish link which I merged separately (since it had been open for quite a long time).

**Comment by vahidhashemian** [comment]
@hachikuji Yes, just saw it this morning. I rebased the PR. The Jenkins error should be unrelated.

File: docs/design.html (hunk 1 above)
Comment (hachikuji) [reply docs/design.html:331]: Seems reasonable.

**Comment by hachikuji** [comment]
LGTM. Thanks again for looking closely at the docs!

**Pull request closed without merging**